definitions/npm/bitcoinjs-lib_v2.x.x/flow_>=v0.17.x/bitcoinjs-lib_v2.x.x.js

```diff
@@ -97,6 +97,8 @@ declare module 'ecurve' {
 }
 // ---------- copypasta end ---- 
 
+import {Point as ECPoint} from 'ecurve';
```

Comment:
I have added more tests for the ECPair constructor, to test if it really checks the parameters with the `ecurve.Point` type, let's see what happens....